src/Microsoft.ML.Transforms/OneHotEncoding.cs

```diff
@@ -141,8 +166,24 @@ internal OneHotEncodingTransformer(ValueToKeyMappingEstimator term, IEstimator<I
         IRowToRowMapper ITransformer.GetRowToRowMapper(DataViewSchema inputSchema) => ((ITransformer)_transformer).GetRowToRowMapper(inputSchema);
     }
     /// <summary>
-    /// Estimator which takes set of columns and produce for each column indicator array.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="OneHotEncodingTransformer"/>. The Estimator takes set of columns and produces an indicator array for each column.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | Text |
+    /// | Output column data type | Vector of floats |
+    /// | Required NuGet in addition to Microsoft.ML | None|
+    ///
+    /// The resulting <see cref="OneHotEncodingTransformer"/> creates a new column, named as specified by the outputColumnName parameter,
```

Comment:
>see [](start = 23, length = 3)

you can't use `<see>` inside markdown should be `<xref:Microsoft.ML.Transforms.OneHotEncodingTransformer>` instead